docs/key_benefits.md

```diff
@@ -0,0 +1,26 @@
+# Key Benefits of `scores`
+
+## Data Handling
+
+- Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
+  - gridded Earth system data (e.g., numerical weather prediction models)
+  - tabular, point, latitude/longitude or site-based data (e.g., forecasts for specific locations).
+- Handles missing data, masking of data and weighting of results.
+- Supports [xarray](https://xarray.dev/) datatypes, and works with [NetCDF4](https://doi.org/10.5065/D6H70CW6), [HDF5](https://github.com/HDFGroup/hdf5), [Zarr](https://zarr.dev) and [GRIB](https://codes.wmo.int/grib2) data sources among others. 
+
+## Usability
+
+- A companion Jupyter Notebook [tutorial](project:./tutorials/Tutorial_Gallery.md) for each metric and statistical test that demonstrates its use in practice.
+- Novel scores not commonly found elsewhere (e.g., [ FIxed Risk Multicategorical (FIRM)](https://doi.org/10.1002/qj.4266), [Flip-Flop Index](https://doi.org/10.1002/met.1732)).
+- Complex scores (e.g., threshold-weighted continuous ranked probability score (twCRPS))
```

Comment:
I added this section (this dot point isn't in the JOSS paper draft), as I thought it might be useful to include here. But, if people think this section should be shorter, I am also happy to remove this dot point.

Reply:
If this dot point stays: @nicholasloveday is twCRPS (a) the correct abbreviation and (b) is it useful to include it here? If I should either amend it, or remove this abbreviation, please let me know.